src/server/network.ts

```diff
@@ -401,6 +417,32 @@ export class Response extends SdkObject {
   frame(): frames.Frame {
     return this._request.frame();
   }
+
+  transferSize(): number | undefined {
+    return this._transferSize;
+  }
+
+  bodySize(): number | undefined {
+    if (!this._transferSize)
+      return;
+    return this._transferSize - this.headersSize();
+  }
+
+  httpVersion(): string {
+    if (!this._httpVersion)
+      return 'HTTP/1.1';
+    if (this._httpVersion === 'http/1.1')
+      return 'HTTP/1.1';
+    return this._httpVersion;
+  }
+
+  headersSize(): number {
+    let rawHeaders = `${this.httpVersion()} ${this.status()} ${this.statusText()}\r\n`;
```

Comment:
ditto.

Unfortunately this is all approximation, I wonder if users are going to be happy about it. It'll be not worse than in DevTools, but different from real network traffic due to headers encoding.